wire-ios-data-model/Source/Model/AccountManager.swift

```diff
@@ -35,7 +35,7 @@ fileprivate extension UserDefaults {
 }
 
 /// Class used to safely access and change stored accounts and the current selected account.
-@objcMembers public final class AccountManager: NSObject {
```

Comment:
Any idea why those marked with @objc prefix?

Reply:
I assume for legacy reasons. I've confirmed that these things are no longer used though from obj-c

Reply:
@agisilaos I'm resolving this comment to merge this PR. If you would like to discuss further feel free to re-open the comment.